ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -406,6 +430,65 @@ protected LayoutParams generateLayoutParams(final ViewGroup.LayoutParams p) {
         return generateDefaultLayoutParams();
     }
 
+    //region Card Bounce Animation
+
+    /**
+     * This method will either create a new bounce Animation or start the one that is already created.
+     */
+    @UiThread
+    private void animateFirstCardView() {
+        if (mActiveCard != null) {
+            return;
+        }
+
```

Comment:
we should also skip bouncing the first card if another animation is running.
```
if (mAnimation != null) {
    return;
}
```

Reply:
Ok